### ISSUE TITLE
Audio Player Overlapping Entire Tabs

### ISSUE THREAD

**Issue by 0xrxL** [libre-tube/LibreTube#4114]
### Steps to reproduce

Make a long press on a video and tap on ```Play in the background```

### Expected behavior

Always available minimizing controls for only audio player.

### Actual behavior

Sometimes audio player cannot be minimized, after minimizing it 1/2 times (or only after 1 time).

https://youtube.com/watch?v=5wXmpt30cBg

### LibreTube version

Last 0.15.1 Debug

### Android version

Android 13 One Ui 5.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by StevenAlexander44** [comment]
> Screen_Recording_20230811_203654.mp4

I get the same issue.

**Comment by Unaiperezdiaz** [comment]
Now it's still on the stable version 0.20.0

**Comment by Bnyro** [comment]
https://github.com/libre-tube/LibreTube/issues/5205
